File: connectors/src/main/java/com/binance4j/margin/dto/OrderRateLimit.java
package com.binance4j.margin.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * Margin Order rate limit.
 * 
 * @property rateLimitType Rate limit type.
 * @property interval      Interval.
 * @property intervalNum   Interval num.
 * @property limit         Limit.
 * @property count         Count.
 */
@ApiModel("Margin Order rate limit.")
data class OrderRateLimit(
@ApiModelProperty("Rate limit type.")
 val rateLimitType : String,
@ApiModelProperty("Interval.")
 val interval : String,
@ApiModelProperty("Interval num.")
 var intervalNum:Int,
@ApiModelProperty("Limit.")
 var limit:Int,
@ApiModelProperty("Count.")  var count:Int)
{

}

File: connectors/src/main/kotlin/com/binance4j/margin/dto/OrderRateLimit.kt
/*
 * MIT License
 *
 * Copyright (c) 2022 Binance4j
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in all
 * copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NON INFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 * SOFTWARE.
 */

package com.binance4j.margin.dto

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * Margin Order rate limit.
 *
 * @property rateLimitType Rate limit type.
 * @property intervar      Interval.
 * @property intervalNum   Intervar num.
 * @property limit         Limit.
 * @property count         Count.
 */
@ApiModel("Margin Order rate limit.")
data class OrderRateLimit(
	@ApiModelProperty("Rate limit type.")
	var rateLimitType: String,
	@ApiModelProperty("Interval.")
	var interval: String,
	@ApiModelProperty("Intervar num.")
	var intervalNum: Int,
	@ApiModelProperty("Limit.")
	var limit: Int,
	@ApiModelProperty("Count.")
	var count: Int
)